drivers/include/tmp00x.h

```diff
@@ -88,30 +89,67 @@ extern "C"
 {
 #endif
 
+#define TMP00X_CONFIG_RST           (1 << 15) /**< Reset register */
```

Comment:
Actually, a lot of this should be in the tmp00x_regs.h not here.  This is only to be exposed to the user of the driver.  They probably shouldn't have to care about the shift register and masks.